tests/analyzer/sourcecode/code-execution.py

```diff
@@ -98,16 +98,13 @@ def get_version():
 
 def RunCustomCommand(command_list):
     print("Running command: %s" % command_list)
-    # ok: code-execution
+    # ruleid: code-execution
     p = subprocess.Popen(command_list, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
 
 
 # ok: code-execution
 subprocess.run(["python3", "-m", "pycodestyle", "."], env=env, check=True)
 
-# ok: code-execution
```

Comment:
same comment here - unless we do some kind of metavariable matching on the variable name (not contents), which might not be worth it. Let's revisit if needed